app/exawind/exawind.cpp

```diff
@@ -118,11 +132,12 @@ int main(int argc, char** argv)
     }
 
     exawind::OversetSimulation sim(MPI_COMM_WORLD);
-    sim.echo(
-        "Initializing AMR-Wind on " + std::to_string(num_awind_ranks) +
-        " MPI ranks");
-    if (amr_comm != MPI_COMM_NULL)
+    if (amr_comm != MPI_COMM_NULL) {
```

Comment:
starting here you start checking `amr_comm` instead of `use_amr_wind`. Want to just use `use_amr_wind` check?

Reply:
yep I can do that!